server/container_create.go

```diff
@@ -62,6 +64,33 @@ func addOciBindMounts(sb *sandbox, containerConfig *pb.ContainerConfig, specgen
 	return nil
 }
 
+func addDevices(sb *sandbox, containerConfig *pb.ContainerConfig, specgen *generate.Generator) error {
+	sp := specgen.Spec()
+	for _, device := range containerConfig.GetDevices() {
+		d, err := devices.DeviceFromPath(device.HostPath, device.Permissions)
```

Comment:
or maybe swap `device` with `d` in the for and that's it :smile: I don't care either way